src/ifcblenderexport/blenderbim/bim/operator.py

```diff
@@ -4316,6 +4316,80 @@ def execute(self, context):
         return {"FINISHED"}
 
 
+class AddToPresentationLayer(bpy.types.Operator):
+    bl_idname = "bim.add_to_presentation_layer"
+    bl_label = "Add To Presentation Layer"
+    index: bpy.props.IntProperty()
+
+    def execute(self, context):
+        presentation_layer = bpy.context.scene.BIMProperties.presentation_layers[self.index]
+
+        # vl = bpy.context.scene.view_layers[presentation_layer.name]
+
+        for obj in bpy.context.selected_objects:
+            obj.BIMObjectProperties.presentation_layer.name = presentation_layer.name
+            elem_name = obj.BIMObjectProperties.attributes["Name"].string_value
+            obj.hide_set(not presentation_layer.layer_on)
+            print(f'Adding "{elem_name}" to View/Presentation Layer "({self.index} - {presentation_layer.name})"')
+
+        return {"FINISHED"}
+
+
+class RemoveFromPresentationLayer(bpy.types.Operator):
```

Comment:
Renamed to "Unassign" instead of "RemoveFrom" to be consistent with the rest of the code.